crates/compiler/builtins/bitcode/src/main.zig

```diff
@@ -164,7 +164,7 @@ comptime {
     exportUtilsFn(utils.decrefCheckNullC, "decref_check_null");
     exportUtilsFn(utils.allocateWithRefcountC, "allocate_with_refcount");
 
-    @export(utils.panic, .{ .name = "roc_builtins.utils." ++ "panic", .linkage = .Weak });
+    @export(utils.panic, .{ .name = "roc_builtins.utils." ++ "panic", .linkage = .Weak }); // zig 0.10 error: TODO: implement exporting with field access
```

Comment:
Might be able to work around this with a `const` declaration.

Reply:
Making a `const` declaration should indeed work:
```zig
const utils_panic = utils.panic;
@export(utils_panic, .{.name = "etc etc"});
```